auth-web/public/config/configuration.json

```diff
@@ -24,5 +24,6 @@
   "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca",
   "LEGAL_API_VERSION": "/api/v1",
   "ONE_STOP_URL": "https://dev.onestop.gov.bc.ca/",
-  "CORPORATE_ONLINE_URL": "https://dev.corponline.gov.bc.ca/"
+  "CORPORATE_ONLINE_URL": "https://dev.corponline.gov.bc.ca/",
+  "FAS_WEB_URL": "https://fas-dev.apps.silver.devops.gov.bc.ca/"
```

Comment:
do we need FAS web url here?

Reply:
yeah to add it in session as main.ts would not be running in library mode